### DISCUSSION TITLE
Mod list in gameplay

### DISCUSSION THREAD

**Discussion by void-v4** [TeamRizu/OutFox#259] [03 - Feature Requests]
Hello!
I'd like it if you added a mod list like [this](https://cdn.discordapp.com/attachments/439809793964769291/734383813941002310/mod_list.png) to the game. It would also be cool if it supported .sm mods, not only .crs mods.

**Comment by Jousway** [comment]
we can add the list using lua but may I ask what .sm mods are

**Comment by mikreo** [comment]
They might mean #ATTACKS in the sm/ssc file.
